schema.sql

```diff
@@ -1,5 +1,20 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+-- Table: VET1.animals
+
+-- DROP TABLE IF EXISTS "VET1".animals;
+
+CREATE TABLE IF NOT EXISTS "VET1".animals
+(
+    id integer NOT NULL,
```

Comment:
:orange_circle: [OPTIONAL] Please make sure your `id` column is autoincremented. You can achieve this by adding the following constraint  IDENTITY(1,1) as shown here  :point_right:  ` id int NOT NULL IDENTITY(1,1)`

In PostgreSQL, IDENTITY(1,1) is used to define a column as an auto-incrementing identity column. This means that the column's values will automatically increase in a sequential manner each time a new row is inserted into the table.

The two numbers within IDENTITY(1,1) have specific meanings:

  - The first number (1 in this case) represents the starting value of the identity column. It indicates the initial value that the column will have when the first row is inserted into the table. In this example, it starts at 1.

  - The second number (also 1 in this case) represents the increment or step value. It specifies how much the column value will increase by each time a new row is added. In this example, it increases by 1, so each new row will have a value that is 1 greater than the previous row.